modules/exploits/linux/upnp/airties_miniupnpd_soap_bof.rb

```diff
@@ -0,0 +1,132 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
```

Comment:
coolio, sounds like an easy way to circumvent :-) I guess you could fork too? but yup, good enough to me.